lnrpc/rpc.proto

```diff
@@ -2147,7 +2147,7 @@ message Payment {
     /// The path this payment took
     repeated string path = 4 [ json_name = "path" ];
 
-    /// The fee paid for this payment in satoshis
+    /// Deprecated, use fee_sat or fee_msat.
```

Comment:
We should also add `deprecated = true` to this field, like we do with the others.